src/main/flight/imu.c

```diff
@@ -723,7 +723,7 @@ void imuUpdateAttitude(timeUs_t currentTimeUs)
 
         // Update the throttle correction for angle and supply it to the mixer
         int throttleAngleCorrection = 0;
-        if (throttleAngleValue && (FLIGHT_MODE(ANGLE_MODE) || FLIGHT_MODE(HORIZON_MODE)) && ARMING_FLAG(ARMED)) {
+        if (throttleAngleValue && (FLIGHT_MODE(ANGLE_MODE | HORIZON_MODE)) && ARMING_FLAG(ARMED)) {
```

Comment:
```suggestion
        if (throttleAngleValue && FLIGHT_MODE(ANGLE_MODE | HORIZON_MODE) && ARMING_FLAG(ARMED)) {
```

Reply:
Can I leave them there just for clarity?

Reply:
Agree with suggestion. `FLIGHT_MODE(ANGLE_MODE | HORIZON_MODE)` does not need another set of parenthesis here.

Reply:
@ctzsnooze : Maybe adding line breaks will help?